hack/ginkgo-e2e.sh

```diff
@@ -102,6 +102,7 @@ export PATH=$(dirname "${e2e_test}"):"${PATH}"
   --repo-root="${KUBE_ROOT}" \
   --node-instance-group="${NODE_INSTANCE_GROUP:-}" \
   --prefix="${KUBE_GCE_INSTANCE_PREFIX:-e2e}" \
+  --os-distro="${OS_DISTRIBUTION:-debian}" \
```

Comment:
Will revise it.

@roberthbailey @ixdy, does jenkins running GKE tests use the env variable OS_DISTRIBUTION by default? If not, @wonderfly may need to add a line to export it in our own GKE jenkins jobs.